.github/workflows/CD.yml

```diff
@@ -14,14 +14,9 @@
 name: C/D
 
 on:        
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'the version number to release'
-        required: true
-      sha:
-        description: 'the github sha to release'
-        required: true
+  push:
+    tags:
```

Comment:
We also need to add a regex to ignore lambda tags

Reply:
Nvm, I got myself confused. This will only execute with tags pushed for this regex correct?

Reply:
Yes this should only execute for tags pushed matching this pattern

Reply:
Does `pkg/v1.23.0/test` trigger this regex?

Reply:
Disregard the last comment.

Reply:
Just tested on my fork and it did not trigger on that regex.  Is that the intended behavior?